service/controller/v20/resource/node/create.go

```diff
@@ -154,3 +193,46 @@ func isPodOfNodeRunning(pods []corev1.Pod, n corev1.Node) bool {
 
 	return false
 }
+
+// removeFromEndpointAddressList is removing slice elements from `addresses` defined by `indexesToRemove`.
+func removeFromEndpointAddressList(addresses []corev1.EndpointAddress, indexesToRemove []int) []corev1.EndpointAddress {
+	var newAddresses []corev1.EndpointAddress
+	for i, ip := range addresses {
+		remove := false
+		for _, j := range indexesToRemove {
+			if i == j {
+				remove = true
+			}
+		}
+		if !remove {
+			newAddresses = append(newAddresses, ip)
+		}
+	}
+	return newAddresses
+}
+
+// removeDeadIPFromEndpoints compares endpoint IPs with current state of nodes and
+// removes any IP addresses that does not belong to any node.
+func (r *Resource) removeDeadIPFromEndpoints(endpoints *corev1.Endpoints, nodes []corev1.Node) (int, *corev1.Endpoints) {
```

Comment:
I'm here to talk about serious stuff. This is a method so should be right after `Resource.create`.

Reply:
I dont follow

Reply:
I assume Pawel was funny and meant something about the method order. A little nitpick. No biggy.